src/main/java/org/springframework/data/hazelcast/repository/support/HazelcastEntityInformation.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2014-2016 the original author or authors.
```

Comment:
We should make it 2018. We can update all headers after the merge...

Reply:
@mesutcelik , yeah, I'd change all headers + adapt code formatting to the Hazelcast style in a separate PR after this one is merged.